.travis.yml

```diff
@@ -39,6 +44,7 @@ before_install:
 install:
     - travis_wait pip install -r requirements.txt
     - pip install sip pyqt5==5.11.*
+    - source $TRAVIS_BUILD_DIR/.travis/install_orange.sh
```

Comment:
you use underscore here, but - in the filename

Reply:
We must be blind. 🙈